QUEST.tsv

```diff
@@ -946,58 +946,58 @@ QUEST_20150317_000945	Eating more would be rather risky.
-QUEST_20150317_000991	X month the 5th {nl} - {nl} I wanted to befriend my new neighbors as soon as possible so I brought some potato dumplings over. {nl}At the door, I could hear the house rattle. My neighbor then appeared, expressionless, {nl}but went back to a cheerful note when he saw the potato dumplings. {Nl}
-QUEST_20150317_000992	He apologized for the lack of spirit before, {nl}as he and his son were putting up the wallpaper. {Nl} What a harmonious family, I wonder if I could create something like that?
-QUEST_20150317_000993	X month the 10th {nl} - {nl} Mr. Raman, who lived at the bank of the creek opposite from us, went on leave.{nl}His wife said he was off to get firewood. {nl} Well, for a long time he went to Klaipeda. {nl}I've heard that you can find a lot of beautiful girls at Klaipeda. {Nl}
-QUEST_20150317_000994	X month the 13th {nl} - {nl} Today was Goddess Origin Day, but somehow everyone in town had stomach pains, {nl}and could not celebrate. Why this happened specifically on this day I do not know. {Nl}Well, illnesses will be illnesses. After all that, we had to make potato porridge. {Nl}
-QUEST_20150317_000995	X month the 14th {nl} - {nl} Last night I had a quarrel with my little sister. {nl}Writing down what you really think isn't that embarrassing. After what happened, {nl}I made a necklace as a present to her, and she came back to me. {nl}Well, I had to give in eventually. Even so, she disappeared off somewhere one morning. {Nl}
-QUEST_20150317_000996	Ah, just now my neighbor made me some delicious porridge. {Nl} In return I made them some warm soup. I can cook too, you know? {Nl}Tonight will be another feast so I'm going to make sure to eat and drink plenty.
-QUEST_20150317_000997	X month the 16th {nl} - {nl} I got a stomach-ache drinking soup that I didn't know already went bad. {Nl} I double-checked the smell before eating it, but there was no bad odor.  {Nl}I felt horrible inside so I went to the well for water. Even with the water, I continued to feel horrible. {Nl}
-QUEST_20150317_000998	I've been throwing up a lot lately but it seems to be getting better now.  {Nl}Tomorrow I'm going to go see Mr. Gordon to get some medicinal herbs. {Nl} My little sister still hasn't come back yet. When I go to her room, I find no clothes. Even Mr. Man is getting worried.
-QUEST_20150317_000999	X month the 17th {nl} - {nl} The village was eerily quiet today.  {Nl}There are no signs of the people who went earlier to get herbs from Mr. Gordon.
-QUEST_20150317_001000	X month the 23rd {nl} - {nl} This might be the last time I get to write in this diary. {Nl} Liliana, you are the only family I have left in this world… {nl} Please be alive somewhere… {Nl}
+QUEST_20150317_000989	X Month, the 4th {nl}- {nl}There's been very little traffic coming to town from the capital. {nl}When one family moved in, the whole village threw a welcome party. {nl}After all, how many outsiders do we get to meet these days?
```

Comment:
In the original it says X월. If it were 1월 it would be January, or it would be October if it said 10월.
I suggest to just remove *Month* and leave *X*. 

So, X, the 4th.